http_server.go

```diff
@@ -36,6 +36,9 @@ func (ats *Suite) StartHttpServer() {
 	// bounce binary response with information in headers
 	mux.HandleFunc("/bounce", bounceBinary)
 
+	// Start listening into proxy
+	ats.HttpServer.Proxy.Listen(mux, "/proxy")
+
 	ats.httpServer = http.Server{
```

Comment:
the basic idea is nice but i am not sure if that is a cool idea and inline with "ats.httpServer". ats.HttpServer should remain the config, and not being used as server item.

so, something like

```
ats.proxyServer = NewProxy(ats.HttpServer.Proxy)
```

would better.